### PR TITLE
Issue #386 Adding workaround in the troubleshooting for error with ...

### PR DESCRIPTION
**Pull request by Preeticp**
…deleting snapshots using virsh.
In addition made a couple of language changes and added a TOC tag to one of the subtitles.

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -97,18 +99,54 @@ manually but this is very **risky**.
...
+
+If you use virsh on KVM/libvirt to create snapshots in your development workflow, using `minishift delete` to delete the snapshots, along with the VM, returns an error:
+
+    $ minishift delete
+    Deleting the Minishift VM...
+    Error deleting the VM:  [Code-55] [Domain-10] Requested operation is not valid: cannot delete inactive domain with 4 snapshots
+
+
+Workaround: The snapshots are stored in `~/.minishift/machines`, but the definitions are stored in `var/lib/libvirt/qemu/snapshot/minishift`.
+To delete the snapshots you need to:
```
Comment (gbraad) [inline docs/troubleshooting.md]: Add whiteline

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -97,18 +99,54 @@ manually but this is very **risky**.
...
+
+### `minishift delete` fails to undefine snapshots of running instances, made using virsh on KVM/libvirt.
+
+If you use virsh on KVM/libvirt to create snapshots in your development workflow, using `minishift delete` to delete the snapshots, along with the VM, returns an error:
+
+    $ minishift delete
+    Deleting the Minishift VM...
+    Error deleting the VM:  [Code-55] [Domain-10] Requested operation is not valid: cannot delete inactive domain with 4 snapshots
+
+
```
Comment (gbraad) [inline docs/troubleshooting.md]: Remove whiteline

**Comment by Preeticp** [comment]
I have taken care of the white lines @gbraad, @hferentschik.

**Comment by praveenkumar** [comment]
Since it approved by respective reviewer, merging now.

**Pull request merged**